client/src/partials/debtors/groups.edit.html

```diff
@@ -12,7 +12,7 @@
   <div class="row">
     <div class="col-md-5">
 
-      <form name="debtorGroup" class="form" autocomplete="off" autocorrect="off" autocapitalize="off">
+      <div class="form" autocomplete="off" autocorrect="off" autocapitalize="off">
```

Comment:
Does `class="form"` actually do anything?  I don't think there is a native bootstrap `form` class unless you wrote it.

Also, should the `autocomplete`, etc, attributes be moved up to the top level `ngForm` directive?